src/controllers/autoCompletion.controller.js

```diff
@@ -5,6 +5,10 @@ exports.getAutoCompletions = async function (req, res, next) {
   const { userInput } = req.query;
   const MAXIMUM_AUTO_COMPLETIONS = 5;
 
+  if (!userInput) {
+    return res.status(200).send([]);
```

Comment:
```suggestion
    return res.status(200).send({
      result: "ok"
      searchHistories: []
    });
```
빈 배열 보다는 객체로 변수에 담아 보내주면 좋을 것 같아요!

밑에 응답 부분도 같은 양식으로 수정하면 좋을 것 같습니다! 
res.status(200).send(searchHistories);

Reply:
네! 객체로 보내는게 더 좋을 것 같네요! 수정하겠습니다!